### ISSUE TITLE
Unable to play recordings.

### ISSUE THREAD

**Issue by GoogleCodeExporter** [google-code-export/mythbox#130]
```
What steps will reproduce the problem?
1. Select a recording and press Play or Play+Skip

What is the expected output? What do you see instead?
The recording plays. 

Error: onClick
argument 1 must be unicode or str

What version of the product are you using? On what operating system?
Dharma 4 OS X 10.6

Please provide any additional information below.
This error is for player.py:162-163 the thumbnailCache.get is not returning the 
correct value.
```

Original issue reported on code.google.com by `hentsche...@gmail.com` on 14 Nov 2010 at 3:41

**Comment by GoogleCodeExporter** [comment]
```
[deleted comment]
```

**Comment by GoogleCodeExporter** [comment]
```
Fixed in changeset 7b625ab8a928
```

Original comment by `semir.pa...@gmail.com` on 15 Nov 2010 at 2:47
